graylog2-web-interface/src/views/components/contexts/FieldTypesProvider.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { useStore } from 'stores/connect';
+import { FieldTypesStore } from 'views/stores/FieldTypesStore';
+import FieldTypesContext from './FieldTypesContext';
+
+const FieldTypesProvider = ({ children }: { children: React.Node }) => {
```

Comment:
We should name this either `DefaultFieldTypesProvider` or `StoreBackedFieldTypesProvider` or something like this, so it's clear that it is actually a specific implementation for the provider and not just the "interface".